spec/serializers/person_serializer_spec.rb

```diff
@@ -94,28 +95,25 @@
     end
 
     before do
-      profile = person.latest_profile
-      profile.profile_identifiers = profile_identifiers
-      profile.save!
+      person.update(criminal_records_office: nil, police_national_computer: 'ABC123456', prison_number: 'XYZ123456')
```

Comment:
maybe we can define a new `let(:person)` with those values for this context instead of an update?